ksctl-components/manifests/versions.go

```diff
@@ -1,10 +1,12 @@
+// Dont edit it unless you know what you are doing
+// This file contains the version of the ksctl agent and ksctl state import
+// used to provide the ksctl core which version to fetch based on ksctl branch
+// KsctlApplicationStackBranchOrTagName CAUTION: make sure it is restored to `latest`
+// Once the dev is done
 package manifests
 
 const (
-	KsctlAgentAppVersion       = "latest"
-	KsctlStateImportAppVersion = "latest"
-
-	// KsctlApplicationStackBranchOrTagName CAUTION: make sure it is restored to `latest`
-	// Once the dev is done
-	KsctlApplicationStackBranchOrTagName = "latest"
+	KsctlAgentAppVersion                 = "feat-ksctl-application-hybrid-client"
+	KsctlStateImportAppVersion           = "feat-ksctl-application-hybrid-client"
+	KsctlApplicationStackBranchOrTagName = "feat-ksctl-application-hybrid-client"
```

Comment:
```suggestion
	KsctlAgentAppVersion                 = "latest"
	KsctlStateImportAppVersion           = "latest"
	KsctlApplicationStackBranchOrTagName = "latest"
```
Also delete the ghcr.io/artifact for this branch